index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Flutter is a mobile app SDK for iOS and Android
+title: Welcome to Flutter
```

Comment:
Usability might be inclusive of "how do people find our docs via search". But no problem, happy to do another PR.